gwy/templates/main.go.tmpl

```diff
@@ -251,10 +254,27 @@ func SetupMux(ctx context.Context, cfg proxyConfig) *http.ServeMux {
 		http.ServeFile(w, r, cfg.swagger)
 	})
 
-	gwmux := runtime.NewServeMux(
-		runtime.WithIncomingHeaderMatcher(incomingHeaderMatcher),
-		runtime.WithOutgoingHeaderMatcher(outgoingHeaderMatcher),
-	)
+	options := []runtime.ServeMuxOption{
+	    runtime.WithIncomingHeaderMatcher(incomingHeaderMatcher),
+	    runtime.WithOutgoingHeaderMatcher(outgoingHeaderMatcher),
+	}
+
+	if !cfg.useJSONNames {
+		marshaller := &runtime.HTTPBodyMarshaler{
+			Marshaler: &runtime.JSONPb{
+				MarshalOptions: protojson.MarshalOptions{
+					UseProtoNames:   true,
```

Comment:
should this be `!cfg.useJSONNames` ?

Reply:
It could be, but I feel that setting exact value is more explicit and easier to read than
```UseProtoNames: !cfg.useJSONNames```

Reply:
I may not understand Go enough.  Is the config value used somewhere else?

Reply:
only to check whether we need a custom marshaller : https://github.com/namely/docker-protoc/pull/222/files#diff-bec7b040a15e52d4cd0b8ffc6c7a0a0b9e95fe7cfb9dc4358b56826043db6ff7R262

Reply:
I see - nevermind - I should have caught that.  I guess this is fine since we're not supporting the ability to not emit unpopulated / customize discard unknown.  Thanks